test/e2e/trust_test.go

```diff
@@ -74,4 +74,18 @@ var _ = Describe("Podman trust", func() {
 		}
 		Expect(teststruct["default"][0]["type"]).To(Equal("insecureAcceptAnything"))
 	})
+
+	It("podman image trust show --json", func() {
+		session := podmanTest.Podman([]string{"image", "trust", "show", "--json"})
+		session.WaitWithDefaultTimeout()
+		Expect(session.ExitCode()).To(Equal(0))
+		Expect(session.IsJSONOutputValid()).To(BeTrue())
+	})
+
+	It("podman image trust show --raw", func() {
+		session := podmanTest.Podman([]string{"image", "trust", "show", "--raw"})
+		session.WaitWithDefaultTimeout()
+		Expect(session.ExitCode()).To(Equal(0))
+		Expect(session.IsJSONOutputValid()).To(BeTrue())
```

Comment:
Can we verify some fields of the output string, like `Expect(session.OutputToString()).To(ContainSubstring("signed"))` above?